src/templates/components/forms_common_rows.html.tt

```diff
@@ -44,6 +44,12 @@
         hint=hint || (no_hint ? '' : l('Kindly provide your complete address.<br>This will be used to authenticate your identity when you open a real account.')) %]
 [% END %]
 
+[% BLOCK address_validation_text %]
+    <span class="gr-12">
+        <p>[% l("You can use letters and numbers (A-Z, a-z, 0-9) and the following special characters: -.'#;:()") %]</p>
```

Comment:
1. I'm not sure mentioning `A-Z, a-z` here is a good idea, as it is allowed to use characters from other languages.
2. To put special characters in placeholder, prevents translation update in case of changing the characters list in future.
`<p>[% l("... and the following special characters: [_1]", "-.'#;:()") %]</p>`

Thanks.

Reply:
For 1) I'll revert back to Ashkan regarding this, as the text to be displayed was based on feedback we received.
I'll make with the changes for 2).